.github/workflows/security-scan.yaml

```
@@ -10,11 +10,14 @@ on:
 jobs:
   python-scans:
     name: Scan Python project
-    uses: canonical/starflow/.github/workflows/scan-python.yaml@main
+    # uses: canonical/starflow/.github/workflows/scan-python.yaml@main
+    uses: lengau/starflow/.github/workflows/scan-python.yaml@work/CRAFT-3707/uv
```

After merging https://github.com/canonical/starflow/pull/27:

```suggestion
    uses: canonical/starflow/.github/workflows/scan-python.yaml@main
```